AutomatedTesting/Gem/PythonTests/EditorPythonTestTools/editor_python_test_tools/editor_entity_utils.py

```diff
@@ -258,6 +259,7 @@ def add_component(self, component_name: str) -> EditorComponent:
         :return: Component object of newly added component.
         """
         component = self.add_components([component_name])[0]
+        self.components.append(component)
```

Comment:
is this going to double append components to the list since add_component() calls add_components() which also calls self.components.append()

Reply:
Good catch, line 284 appears to also modify self.components before returning a list of new components.

Reply:
Nice catch. Fixed.

Reply:
> 
> 
> Good catch, line 284 appears to also modify self.components before returning a list of new components.

Needs to modify self.components for reference via component index, e.g.:

```
spawner_entity.components[2].get_component_property_value("Controller|Configuration|Mesh Asset")
```

but this helper needs to return only the added components, not all of them.